Package.swift

```diff
@@ -191,6 +191,10 @@ let package = Package(
             name: "SWBUniversalPlatform",
             dependencies: ["SWBCore"],
             swiftSettings: swiftSettings),
+        .target(
+            name: "SWBWebAssemblyPlatform",
+            dependencies: ["SWBCore"],
+            swiftSettings: swiftSettings),
```

Comment:
Could you please rebase this and change `swiftSettings` to `swiftSettings(.v6)`? I made some build changes that interfered, sorry